src/ImsStorage/ImsCache.php

```diff
@@ -4,6 +4,9 @@
 
 use Packback\Lti1p3\Interfaces\ICache;
 
+/**
+ * @deprecated
+ */
```

Comment:
Both of these classes are leftover from the old library. I left them in originally as examples, but frankly, they're a mess and won't really function in a modern web app. I'd rather leave it up to people to create their own implementations.